staging/src/k8s.io/apiserver/pkg/endpoints/discovery/aggregated/handler.go

```diff
@@ -36,6 +36,15 @@ import (
 	"k8s.io/klog/v2"
 )
 
+// GroupVersions from the default source take precedence over all others
+type Source uint
+
+const (
+	AggregatorSource Source = 0
```

Comment:
Should this use iota?

Reply:
and e.g. leave gaps if there is any chance people will use constants not from this file